libkbfs/block_types.go

```diff
@@ -294,13 +360,22 @@ type FileBlock struct {
 	hash *kbfshash.RawDefaultHash
 }
 
+var _ BlockWithPtrs = (*FileBlock)(nil)
+
 // NewFileBlock creates a new, empty FileBlock.
 func NewFileBlock() Block {
 	return &FileBlock{
 		Contents: make([]byte, 0, 0),
 	}
 }
 
+// NewFileBlockWithPtrs creates a new, empty FileBlock.
+func NewFileBlockWithPtrs(isInd bool) BlockWithPtrs {
+	fb := NewFileBlock().(*FileBlock)
+	fb.IsInd = isInd
```

Comment:
Is it possible to make a `BlockWithPtrs` that isn't indirect? I don't see this parameter used.

Reply:
Yes, it's used here: https://github.com/keybase/kbfs/pull/1736/files#diff-27199ce3c4667e207829d68cf8646446R543

Like I said above, any dir/file block can be a `BlockWithPtrs`, and here it's used to create a new right-most leaf block for the block tree.

Reply:
OK so we're talking about any block that can be converted to an indirect block.

Reply:
Yes, right.

Reply:
All right let's keep it. Not a big deal anyway.